src/landOnFunctions.js

```diff
@@ -1,57 +1,67 @@
-export {standardProperty, railroad, utility, eventCard, noEvent, go, goToJail, incomeTax, luxuryTax};
+export {
+    standardProperty,
+    railroad,
+    utility,
+    eventCard,
+    noEvent,
+    go,
+    goToJail,
+    incomeTax,
+    luxuryTax,
+};
 
 function standardProperty(activePlayer, emitter){
-  if(this.ownerID == null){
-    emitter.emit(`promptBuy${activePlayer.socketID}`, this, activePlayer);
-  }
-  else{
-    emitter.emit(`promptPayment${activePlayer.socketID}`, this.ownerID, activePlayer, this.rent[houses]);
-  }
+    if(this.ownerID == null){
```

Comment:
It's kind of weird since every other part of the class is laid out in the JSON file.  I have thought about doing what you said, then you have this class with 8 functions that don't apply to them.  Maybe I'm too hesitant to pass too many arguments in sometimes.